tests/browser/test/mutator_test.js

```diff
@@ -0,0 +1,89 @@
+/**
+ * @license
+ * Copyright 2023 Google LLC
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/**
+ * @fileoverview Node.js script to run Automated tests in Chrome, via webdriver.
+ */
+
+const chai = require('chai');
+const {
+  testSetup,
+  testFileLocations,
+  getBlockTypeFromCategory,
+  switchRTL,
+  dragBlockTypeFromFlyout,
+  screenDirection,
+} = require('./test_setup');
+
+let browser;
+suite('Testing Mutator', function (done) {
+  // Setting timeout to unlimited as the webdriver takes a longer time to run than most mocha test
+  this.timeout(0);
+
+  // Setup Selenium for all of the tests
+  suiteSetup(async function () {
+    browser = await testSetup(testFileLocations.playground);
+  });
+
+  test('Testing Field Edits LTR', async function () {
+    await testingMutator(screenDirection.LTR);
+  });
+
+  test('Testing Field Edits RTL', async function () {
+    await switchRTL(browser);
+    await testingMutator(screenDirection.RTL);
+  });
+
+  // Teardown entire suite after test are done running
+  suiteTeardown(async function () {
+    await browser.deleteSession();
+  });
+});
+
+async function testingMutator(delta) {
+  // Drag out print from flyout.
+  const controlIfFlyout = await dragBlockTypeFromFlyout(
+    browser,
+    'Logic',
+    'controls_if',
+    delta * 50,
+    50
+  );
+  // Get the number of transform elements on the if do block
+  const ifDo = await browser.$(
+    '#blocklyDiv > div > svg.blocklySvg > g > g.blocklyBlockCanvas > g.blocklyDraggable.blocklySelected'
+  );
+  let ifDoNumHTML = await ifDo.getHTML();
+  const ifDoTransformCountBefore = (ifDoNumHTML.match(/transform/g) || [])
```

Comment:
I wonder if there is a better thing to assert on than this? it's pretty abstract. it wouldn't be obvious to me if this failed what should have happened or what is broken. I think a better test would be to use the Blockly APIs to check that the block has more inputs now (if you add an else block in the mutator then the final block should have another statement input)